File: output/StarCoder/codenet/C++/Python/s573429823.py
n = int(raw_input())
if n<4:
    print "No"
else:
    for i in range(n/4+1):
        for j in range(n/7+1):
            if i*4+j*7 == n:
                print "Yes"
                break
        else:
            continue
        break
    else:
        print "No"

File: output/StarCoder/codenet/C++/Java/s573429823.java
import java.util.*;
public class s573429823 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        long n = sc.nextLong();
        if( n<4 ){
            System.out.println("No");
        }else{
            for( int i=0; i<=(n/4); i++)
            {
                for( int j=0; j<=(n/7); j++ )
                {
                    if( (i*4+j*7)==n )
                    {
                        System.out.println("Yes");
                        return;
                    }
                }
            }
            System.out.println("No");
        }
    }
}